Makefile

```diff
@@ -1,86 +1,16 @@
-BUILD=_build
+build:
+	dune build
 
-# By default (if left blank), the generator script searches for
-# the opencv4 headers in /usr/include/ and /usr/local/include/.
-# Set this variable if your opencv4 headers are located elsewhere.
-SYSTEM_INCLUDE_DIR=
+install:
+	dune install
 
-GEN=gen
-GENERATOR=generator.py
-GENERATOR_SRC=$(GENERATOR) hdr_parser.py type_manager.py incl/*
+uninstall:
+	dune uninstall
 
-CPP=$(GEN)/opencv.cpp glue.cpp
-HEADERS=$(GEN)/opencv.h glue.h
-SHARED_LIB=libocamlopencv.so
-SHARED_LIBS_INSTALL_DIR=/usr/lib/
-INSTALLED_SHARED_LIB=$(SHARED_LIBS_INSTALL_DIR)/$(SHARED_LIB)
+run:
+	dune run
 
-MLS=$(GEN)/opencv
-FLAGS=-use-ocamlfind
+doc:
```

Comment:
Note that in the README, the Makefile target is declared as `docs`. I do prefer `doc` though, so I'd rather change the README instead.